### PR TITLE
fix(docs): invalid yaml in example

### LINKED ISSUE #28197: Invalid YAML example
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Syntax for GitHub's form schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)

### What part(s) of the article would you like to see updated?

I'd like to see an updated YAML example in [About GitHub's form schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#about-githubs-form-schema) (Example \#1)

The reason why it should be changed is that, the current YAML example is invalid. It shows this error:

```
Nested mappings are not allowed in compact mappings at line 5, column 18
```

The expected outcome is to contain a valid YAML example.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by priyanujgogoi-28**
### Why:

To fix the invalid YAML (example \#1) in [About GitHub's form schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#about-githubs-form-schema) which shows an error:

```
Nested mappings are not allowed in compact mappings at line 5, column 18
```

Added double-quotes to YAML `placeholder` key's value to interpret the colon before `Example` text as literal value.

Closes: #28197

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added double-quotes to the value of `placeholder` YAML key.

```diff
- placeholder: Example: macOS Big Sur
+ placeholder: "Example: macOS Big Sur"
```

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@priyanuj-gogoi Thank you for opening a PR and linking it to your issue! ✨ 

I'll get this triaged for review! :zap:

**Review by subatoi — APPROVED** [review]
Thank you for spotting this and submitting a fix ✨

**Review by Adawntoremember — COMMENTED** [review]
Thanks for stirring me in the right direction.

File: content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ For example, the following form definition includes four form elements: a text a
   attributes:
     label: Operating System
```
Comment (Adawntoremember) [inline content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.md:29]: I guess it is correct now

**Review by Adawntoremember — COMMENTED** [review]
Thank you for the help and support

**Pull request merged**